src/coreclr/inc/patchpointinfo.h

```diff
@@ -57,32 +57,32 @@ struct PatchpointInfo
         m_securityCookieOffset = original->m_securityCookieOffset;
         m_monitorAcquiredOffset = original->m_monitorAcquiredOffset;
 
-        for (unsigned i = 0; i < original->m_numberOfLocals; i++)
+        for (uint32_t i = 0; i < original->m_numberOfLocals; i++)
         {
             m_offsetAndExposureData[i] = original->m_offsetAndExposureData[i];
         }
     }
 
-    // Total size of this patchpoint info record, in bytes
-    unsigned PatchpointInfoSize() const
+    // Total size of this patchpoint32_t info record, in bytes
```

Comment:
This change in the comment seems odd, is `patchpoint32_t` the name of a type somewhere?